### PR TITLE
feat(addressvalidation): start generating apiv1

### PR DESCRIPTION
**Pull request by quartzmo**

### CODE REVIEW

File: addressvalidation/go.mod [hunk 1]
```diff
@@ -0,0 +1,26 @@
+module cloud.google.com/go/addressvalidation
```
Comment (codyoss) [inline addressvalidation/go.mod:1]: Same as the other PR, I think we should group these under a maps module.

**Review by codyoss — CHANGES_REQUESTED** [review]

File: addressvalidation/go.mod (hunk 1 above)
Comment (quartzmo) [reply addressvalidation/go.mod:1]: Agreed, will do.

File: maps/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Address Validation API
+
+[![Go Reference](https://pkg.go.dev/badge/cloud.google.com/go/maps.svg)](https://pkg.go.dev/cloud.google.com/go/maps)
+
+Go Client Library for Address Validation API.
```
Comment (codyoss) [inline maps/README.md:5]: This should be more generic for maps

**Review by codyoss — CHANGES_REQUESTED** [review]
Just one small thing.

File: maps/README.md (hunk 2 above)
Comment (quartzmo) [reply maps/README.md:5]: Will do.

Comment (quartzmo) [reply maps/README.md]: Fixed, thanks!

**Review by codyoss — APPROVED** [review]

**Pull request merged**